samples/snabbdom/counter/webpack.config.js

```diff
@@ -0,0 +1,59 @@
+var path = require("path");
+var webpack = require("webpack");
+
+var cfg = {
+  devtool: "source-map",
+  entry: "./out/app.js",
+  output: {
+    path: path.join(__dirname, "public"),
+    publicPath: "/public",
+    filename: "bundle.js"
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: "source-map-loader",
+        enforce: "pre"
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        options: {
+          presets: [["es2015", {"modules" : false}]],
+          plugins: ["transform-runtime"]
+        },
+      }
+    ]
+  },
+  resolve: {
+    modules: [
+      "node_modules", path.resolve("../node_modules/")
+    ]
+  }
+};
+
+if (process.env.WEBPACK_DEV_SERVER) {
+  cfg.entry = [
+    "webpack-dev-server/client?http://localhost:8080",
+    'webpack/hot/only-dev-server',
+    "./out"
+  ];
+  cfg.plugins = [
+    new webpack.HotModuleReplacementPlugin()
+  ];
+  cfg.module.loaders = [{
+    test: /\.js$/,
+    exclude: /node_modules/,
+    loader: "react-hot-loader"
```

Comment:
The react-hot-loader is continuously changing but the last time I checked react-hot-loader was being referenced in the entry and in the `.babelrc` file, not in the module.loaders/rules section. You can check [this sample](https://github.com/fable-compiler/hmr-sample/blob/master/webpack.config.js) for reference which is based in latest react-hot-boilerplate (which uses react-hot-loader 3, still in beta). 

BTW, I need to test if react-hot-loader can be used with elmish. Apparently, besides the configuration you need to put your components in a file different from the entry file and include [some special code](https://github.com/fable-compiler/hmr-sample/blob/8a70d890f70946b530f2fe44b11a697cbe5e566b/src/main.fsx#L20-L23), that I'd like to make more subtle (probably with inlining):